sdk/tests/conformance2/vertex_arrays/vertex-array-object.html

```diff
@@ -476,17 +476,22 @@
     // delete the color buffers AND the position buffer.
     gl.bindVertexArray(null);
     for (var ii = 0; ii < colorBuffers.length; ++ii) {
-      gl.deleteBuffer(colorBuffers[ii]);
-      gl.deleteBuffer(elementBuffers[ii]);
       gl.bindVertexArray(vaos[ii]);
-      var boundBuffer = gl.getVertexAttrib(1, gl.VERTEX_ATTRIB_ARRAY_BUFFER_BINDING);
+
+      gl.deleteBuffer(colorBuffers[ii]);
+      gl.deleteBuffer(positionBuffer);
```

Comment:
When a buffer, texture, or renderbuffer object is deleted, it is unbound from any
bind points it is bound to in the current context. Since you move deleteBuffer after bindVertexArray, these buffers are  unbound from VAO. You cannot use these buffers below. See https://www.khronos.org/registry/gles/specs/3.0/es_spec_3.0.4.pdf, page 315 "D.1.2 Automatic Unbinding of Deleted Objects".